packages/cfd/build/constants.js

```diff
@@ -14,7 +14,7 @@ const { WebpackManifestPlugin } = require('webpack-manifest-plugin');
 const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
 
 const {
-    copyConfig,
+    // copyConfig,
```

Comment:
@iman-fs Are these changes excepted in this PR? 🤔

Reply:
No . But after I updated this branch with  `develop` branch, eslint complains about some unused codes in this file. since I was not sure if we need them or not, I just commented them out.